graphiti_core/driver.py

```
@@ -0,0 +1,207 @@
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import logging
+from abc import ABC, abstractmethod
+from typing import Any, Coroutine
```

Ruff (0.11.9)

Import from `collections.abc` instead: `Coroutine`

Import from `collections.abc`

(UP035)